google-cloud-core/src/main/java/com/google/cloud/AsyncPage.java

```diff
@@ -17,6 +17,7 @@
 package com.google.cloud;
 
 import com.google.api.gax.core.ApiFuture;
+import com.google.api.gax.core.Page;
```

Comment:
Shouldn't the `AsyncPage` here be pulled out and replaced with the new `AsyncPage` from gax?